### DISCUSSION TITLE
cannot boot from CD

### DISCUSSION THREAD

**Discussion by ntt77** [utmapp/UTM#6663] [General]
I create a new vm with x86_64 architecture and mount the windows 11 IOT LTSC 2024 to that VM.
When booting up, the screen says "press any key to boot from CD/DVD", however, I press keys and nothing happens. And it boot to EFI Shell.

I have macos 12, latest UTM version.

**Comment by Glyx** [comment]
Hi

>***Five — Today at 7:38PM***
>Did you read about the requirement for SSE2 and extended AVX support in 24H2 and onward? 
>That's why it doesn't boot
>UTM can emulate this but the bootloader isn't contempt entirely here 
>
>That will be fixed once Osy has updated the QEMU backend 
>
>
>!! Fair warning !! 
>Emulation is about 40x slower than virtualization 
>
>So please tell us *why* you absolutely must emulate an x86 windows

Git this from the UTM discord server
